tstest/natlab/firewall.go

```diff
@@ -43,33 +84,28 @@ func (f *Firewall) timeNow() time.Time {
 	return time.Now()
```

Comment:
Aside: later we'll want to stop using time.Now (and timers and AfterFunc, etc) and inject clocks, etc so we can fake time entirely and run big long timeout tests in milliseconds.

Reply:
Yeah, I've been careful to try plumbing TimeNow everywhere, but we're going to need a more elaborate fake time that also implements timers and sleeping fairly soon. Maybe in tstime?